### PR TITLE
feat: add strided interface for Weibull distribution

### LINKED ISSUE #946: [RFC]: Add `@stdlib/random/strided/weibull`
### Description

This RFC proposes adding support for filling a strided array with pseudorandom numbers drawn from a Weibull distribution. This is the strided interface analog of `@stdlib/random/base/weibull`.

Package: `@stdlib/random/strided/weibull`
Alias: `weibull`

### Related Issues

None.

### Questions

No.

### Other

- A package with a similar implementation/API design is [`@stdlib/random/strided/beta`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/random/strided/beta).

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by amitjimiwal**
Resolves #946 

## Related Issues
None

This pull request:

-   resolves #946 

## Questions

> Any questions for reviewers of this pull request?

Do lemme know if i interpreted wrong implementation

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/random/strided/weibull/package.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@stdlib/random/strided/weibull",
+  "version": "0.0.0",
+  "description": "Fill a strided array with pseudorandom numbers drawn from a weibull distribution.",
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/package.json]:
```suggestion
  "description": "Fill a strided array with pseudorandom numbers drawn from a Weibull distribution.",
```

File: lib/node_modules/@stdlib/random/strided/weibull/README.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Weibull Random Numbers
+
+> Fill a strided array with pseudorandom numbers drawn from a [weibull][@stdlib/random/base/weibull] distribution.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/README.md]:
```suggestion
> Fill a strided array with pseudorandom numbers drawn from a [Weibull][@stdlib/random/base/weibull] distribution.
```

File: lib/node_modules/@stdlib/random/strided/weibull/README.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+
+## Usage
+
+```javascript
+var weibull = require( '@stdlib/random/strided/weibull' );
+```
+
+#### beta( N, k, sa, lambda, sb, out, so\[, options] )
+
+Fills a strided array with pseudorandom numbers drawn from a [weibull][@stdlib/random/base/weibull] distribution.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/README.md]:
```suggestion
Fills a strided array with pseudorandom numbers drawn from a [Weibull][@stdlib/random/base/weibull] distribution.
```

File: lib/node_modules/@stdlib/random/strided/weibull/README.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+
+// Fill the array with pseudorandom numbers:
+weibull( out.length, [ 2.0 ], 0, [ 5.0 ], 0, out, 1 );
+```
+
+The function has the following parameters:
+
+-   **N**: number of indexed elements.
+-   **k**: scale parameter.
+-   **sa**: index increment for `alpha`.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/README.md]:
```suggestion
-   **sk**: index increment for `k`.
```

Let's make sure to rename the `sa` parameter everywhere to `sk`.

File: lib/node_modules/@stdlib/random/strided/weibull/README.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+-   **lambda**:shape parameter.
+-   **sb**: index increment for `beta`.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/README.md]:
```suggestion
-   **sl**: index increment for `lambda`.
```

Let's make sure to rename the `sb` parameter everywhere to `sl`.

File: lib/node_modules/@stdlib/random/strided/weibull/README.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+var out = new Float64Array( 10 );
+
+// Fill the array with pseudorandom numbers:
+weibull.ndarray( out.length, [ 2.0 ], 0, 0, [ 5.0 ], 0, 0, out, 1, 0 );
+```
+
+The function has the following additional parameters:
+
+-   **oa**: starting index for `k`.
+-   **ob**: starting index for `alpha`.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/README.md]:
```suggestion
-   **ol**: starting index for `lambda`.
```

File: lib/node_modules/@stdlib/random/strided/weibull/README.md [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+// Create an array:
+var out = new Float64Array( 10 );
+
+// Fill the array with pseudorandom numbers:
+weibull.ndarray( out.length, [ 2.0 ], 0, 0, [ 5.0 ], 0, 0, out, 1, 0 );
+```
+
+The function has the following additional parameters:
+
+-   **oa**: starting index for `k`.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/README.md]:
```suggestion
-   **ok**: starting index for `k`.
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+
+// MAIN //
+
+/**
+* Fills a strided array with pseudorandom numbers drawn from a weibull distribution.
+*
+* @param {NonNegativeInteger} N - number of indexed elements
+* @param {Collection} k - scale parameter
+* @param {integer} sa - `k` stride length
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js]:
```suggestion
* @param {integer} sk - `k` stride length
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+* @param {NonNegativeInteger} oa - starting `k` index
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js]:
```suggestion
* @param {NonNegativeInteger} ok - starting `k` index
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+* @param {Collection} lambda - shape parameter
+* @param {integer} sb - `lambda` stride length
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js]:
```suggestion
* @param {integer} sl - `lambda` stride length
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+* @param {NonNegativeInteger} ob - starting `lambda` index
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js]:
```suggestion
* @param {NonNegativeInteger} ol - starting `lambda` index
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/main.js [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+
+
+// MAIN //
+
+/**
+* Fills a strided array with pseudorandom numbers drawn from a beta distribution.
+*
+* @param {NonNegativeInteger} N - number of indexed elements
+* @param {Collection} k - scale parameter
+* @param {integer} sa - `k` stride length
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/main.js]:
```suggestion
* @param {integer} sk - `k` stride length
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/main.js [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+* @param {Collection} lambda - shape parameter
+* @param {integer} sb - `lambda` stride length
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/main.js]:
```suggestion
* @param {integer} sk - `lambda` stride length
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 14]
```diff
@@ -0,0 +1,152 @@
...
+/**
+* Interface describing `weibull`.
+*/
+interface Routine {
+	/**
+	* Fills a strided array with pseudorandom numbers drawn from a weibull distribution.
+	*
+	* @param N - number of indexed elements
+	* @param k - scale parameter
+	* @param sa - `alpha` stride length
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
	* @param sk - `k` stride length
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 15]
```diff
@@ -0,0 +1,152 @@
...
+	*/
+	copy?: boolean;
+}
+
+/**
+* Interface describing `weibull`.
+*/
+interface Routine {
+	/**
+	* Fills a strided array with pseudorandom numbers drawn from a weibull distribution.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
	* Fills a strided array with pseudorandom numbers drawn from a weibull distribution.
```
```suggestion
	* Fills a strided array with pseudorandom numbers drawn from a Weibull distribution.
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 16]
```diff
@@ -0,0 +1,152 @@
...
+	*
+	* @param N - number of indexed elements
+	* @param k - scale parameter
+	* @param sa - `alpha` stride length
+	* @param lambda - shape parameter
+	* @param sb - `weibull` stride length
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
	* @param sl - `lambda` stride length
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 17]
```diff
@@ -0,0 +1,152 @@
...
+	* // Create an array:
+	* var out = new Float64Array( 10 );
+	*
+	* // Fill the array with pseudorandom numbers:
+	* weibull( out.length, [ 2.0 ], 0, [ 5.0 ], 0, out, 1 );
+	*/
+	( N: number, k: Collection, sa: number, lambda: Collection, sb: number, out: Collection, so: number, options?: Options ): Collection; // tslint:disable-line:max-line-length
+
+	/**
+	* Fills a strided array with pseudorandom numbers drawn from a weibull distribution using alternative indexing semantics.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
	* Fills a strided array with pseudorandom numbers drawn from a Weibull distribution using alternative indexing semantics.
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 18]
```diff
@@ -0,0 +1,152 @@
...
+	*
+	* @param N - number of indexed elements
+	* @param k - scale parameter
+	* @param sa - `k` stride length
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
	* @param sk - `k` stride length
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 19]
```diff
@@ -0,0 +1,152 @@
...
+	* @param oa - starting index for `k`
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
	* @param ok - starting index for `k`
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 20]
```diff
@@ -0,0 +1,152 @@
...
+	* @param lambda - shape parameter
+	* @param sb - `lambda` stride length
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
	* @param sl - `lambda` stride length
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 21]
```diff
@@ -0,0 +1,152 @@
...
+	* @param ob - starting index for `lambda`
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
	* @param ol - starting index for `lambda`
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 22]
```diff
@@ -0,0 +1,152 @@
...
+	* @example
+	* var Float64Array = require( `@stdlib/array/float64` );
+	*
+	* // Create an array:
+	* var out = new Float64Array( 10 );
+	*
+	* // Fill the array with pseudorandom numbers:
+	* weibull.ndarray( out.length, [ 2.0 ], 0, 0, [ 5.0 ], 0, 0, out, 1, 0 );
+	*/
+	ndarray( N: number, k: Collection, sa: number, oa: number, lambda: Collection, sb: number, ob: number, out: Collection, so: number, oo: number, options?: Options ): Collection; // tslint:disable-line:max-line-length
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
	ndarray( N: number, k: Collection, sk: number, ok: number, lambda: Collection, sl: number, ol: number, out: Collection, so: number, oo: number, options?: Options ): Collection; // tslint:disable-line:max-line-length
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 23]
```diff
@@ -0,0 +1,152 @@
...
+}
+
+/**
+* Fills a strided array with pseudorandom numbers drawn from a weibull distribution.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
* Fills a strided array with pseudorandom numbers drawn from a Weibull distribution.
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 24]
```diff
@@ -0,0 +1,152 @@
...
+*
+* @param N - number of indexed elements
+* @param k - scale parameter
+* @param sa - `k` stride length
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
* @param sk - `k` stride length
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 25]
```diff
@@ -0,0 +1,152 @@
...
+* @param lambda -  shape parameter
+* @param sb - `lambda` stride length
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
* @param sl - `lambda` stride length
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts [hunk 26]
```diff
@@ -0,0 +1,152 @@
...
+	* @example
+	* var Float64Array = require( `@stdlib/array/float64` );
+	*
+	* // Create an array:
+	* var out = new Float64Array( 10 );
+	*
+	* // Fill the array with pseudorandom numbers:
+	* weibull( out.length, [ 2.0 ], 0, [ 5.0 ], 0, out, 1 );
+	*/
+	( N: number, k: Collection, sa: number, lambda: Collection, sb: number, out: Collection, so: number, options?: Options ): Collection; // tslint:disable-line:max-line-length
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/types/index.d.ts]:
```suggestion
	( N: number, k: Collection, sk: number, lambda: Collection, sl: number, out: Collection, so: number, options?: Options ): Collection; // tslint:disable-line:max-line-length
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 27]
```diff
@@ -0,0 +1,175 @@
+
+{{alias}}( N, k, sa, lambda, sb, out, so[, options] )
+    Fills a strided array with pseudorandom numbers drawn from a weibull
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
    Fills a strided array with pseudorandom numbers drawn from a Weibull
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 28]
```diff
@@ -0,0 +1,175 @@
+
+{{alias}}( N, k, sa, lambda, sb, out, so[, options] )
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
{{alias}}( N, k, sk, lambda, sl, out, so[, options] )
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 29]
```diff
@@ -0,0 +1,175 @@
...
+
+    Parameters
+    ----------
+    N: integer
+        Number of indexed elements.
+
+    k: ArrayLikeObject
+        scale parameter.
+
+    sa: integer
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
    sk: integer
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 30]
```diff
@@ -0,0 +1,175 @@
...
+        Index increment for `alpha`.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
        Index increment for `k`.
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 31]
```diff
@@ -0,0 +1,175 @@
...
+
+    lambda: ArrayLikeObject
+        shape parameter.
+
+    sb: integer
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
    sl: integer
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 32]
```diff
@@ -0,0 +1,175 @@
...
+        Index increment for `beta`.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
        Index increment for `lambda`.
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 33]
```diff
@@ -0,0 +1,175 @@
...
+
+    // Advanced indexing:
+    > a = {{alias:@stdlib/array/linspace}}( 1.0, 5.0, 6 );
+    > b = {{alias:@stdlib/array/linspace}}( 1.0, 5.0, 6 );
+    > out = {{alias:@stdlib/array/zeros}}( 6, 'generic' );
+    > {{alias}}( 3, a, -2, b, 1, out, 1 )
+    [...]
+
+
+{{alias}}.ndarray( N, k, sa, oa, lambda, sb, ob, out, so, oo[, options] )
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
{{alias}}.ndarray( N, k, sk, ok, lambda, sl, ol, out, so, oo[, options] )
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 34]
```diff
@@ -0,0 +1,175 @@
...
+
+    Parameters
+    ----------
+    N: integer
+        Number of indexed elements.
+
+    k: ArrayLikeObject
+        scale parameter.
+
+    sa: integer
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
    sk: integer
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 35]
```diff
@@ -0,0 +1,175 @@
...
+        Index increment for `alpha`.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
        Index increment for `k`.
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 36]
```diff
@@ -0,0 +1,175 @@
...
+
+    oa: integer
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
    ok: integer
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 37]
```diff
@@ -0,0 +1,175 @@
...
+        Starting index for `alpha`.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
        Starting index for `k`.
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 38]
```diff
@@ -0,0 +1,175 @@
...
+
+    lambda: ArrayLikeObject
+        shape parameter.
+
+    sb: integer
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
    sl: integer
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 39]
```diff
@@ -0,0 +1,175 @@
...
+        Index increment for `beta`.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
        Index increment for `lambda`.
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 40]
```diff
@@ -0,0 +1,175 @@
...
+
+    ob: integer
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
    ol: integer
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 41]
```diff
@@ -0,0 +1,175 @@
...
+        Starting index for `beta`.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
        Starting index for `lambda`.
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/index.js [hunk 42]
```diff
@@ -0,0 +1,63 @@
...
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/**
+* Fill a strided array with pseudorandom numbers drawn from a weibull distribution.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/index.js]:
```suggestion
* Fill a strided array with pseudorandom numbers drawn from a Weibull distribution.
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/main.js [hunk 43]
```diff
@@ -0,0 +1,72 @@
...
+
+var nullary = require( '@stdlib/strided/base/nullary' );
+var binary = require( '@stdlib/strided/base/binary' );
+var prng = require( './prng.js' );
+
+
+// MAIN //
+
+/**
+* Fills a strided array with pseudorandom numbers drawn from a beta distribution.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/main.js]:
```suggestion
* Fills a strided array with pseudorandom numbers drawn from a Weibull distribution.
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js [hunk 44]
```diff
@@ -0,0 +1,75 @@
...
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+*
+* // Create an array:
+* var out = new Float64Array( 10 );
+*
+* // Fill the array with pseudorandom numbers:
+* weibull( out.length, [ 2.0 ], 0, 0, [ 5.0 ], 0, 0, out, 1, 0 );
+*/
+function weibull( N, k, sa, oa, lambda, sb, ob, out, so, oo, options ) { // eslint-disable-line max-params
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js]:
```suggestion
function weibull( N, k, sk, ok, lambda, sl, ol, out, so, oo, options ) { // eslint-disable-line max-params
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js [hunk 45]
```diff
@@ -0,0 +1,75 @@
...
+	var rand = prng( k, sa, oa, lambda, sb, ob, arguments.length > 10, options ); // eslint-disable-line max-len
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js]:
```suggestion
	var rand = prng( k, sk, ok, lambda, sl, ol, arguments.length > 10, options ); // eslint-disable-line max-len
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js [hunk 46]
```diff
@@ -0,0 +1,75 @@
...
+	if ( rand.arity === 0 ) {
+		nullary( [ out ], [ N ], [ so ], [ oo ], rand.fcn );
+		return out;
+	}
+	binary( [ k, lambda, out ], [ N ], [ sa, sb, so ], [ oa, ob, oo ], rand.fcn ); // eslint-disable-line max-len
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js]:
```suggestion
	binary( [ k, lambda, out ], [ N ], [ sk, sl, so ], [ ok, ol, oo ], rand.fcn ); // eslint-disable-line max-len
```

**Review by Planeshifter — CHANGES_REQUESTED** [review]
Thanks for your PR!

Flagged a bunch of copy-paste and minor naming issues. Once these are addressed (easiest may be to batch commit the suggestions from the files tab), should be able to merge in.

File: lib/node_modules/@stdlib/random/strided/weibull/README.md [hunk 47]
```diff
@@ -0,0 +1,226 @@
...
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var weibull = require( '@stdlib/random/strided/weibull' );
+```
+
+#### beta( N, k, sa, lambda, sb, out, so\[, options] )
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/README.md]:
```suggestion
#### weibull( N, k, sk, lambda, sl, out, so\[, options] )
```

File: lib/node_modules/@stdlib/random/strided/weibull/README.md [hunk 48]
```diff
@@ -0,0 +1,226 @@
...
+
+var opts = {
+    'seed': 12345
+};
+
+var out = new Float64Array( 10 );
+weibull( out.length, [ 2.0 ], 0, [ 5.0 ], 0, out, 1, opts );
+```
+
+#### weibull.ndarray( N, k, sa, oa, lambda, sb, ob, out, so, oo\[, options] )
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/README.md]:
```suggestion
#### weibull.ndarray( N, k, sk, ok, lambda, sl, ol, out, so, oo\[, options] )
```

File: lib/node_modules/@stdlib/random/strided/weibull/README.md [hunk 49]
```diff
@@ -0,0 +1,226 @@
...
+
+Fills a strided array with pseudorandom numbers drawn from a [weibull][@stdlib/random/base/weibull] distribution using alternative indexing semantics.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/README.md]:
```suggestion
Fills a strided array with pseudorandom numbers drawn from a [Weibull][@stdlib/random/base/weibull] distribution using alternative indexing semantics.
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/main.js [hunk 50]
```diff
@@ -0,0 +1,72 @@
...
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+*
+* // Create an array:
+* var out = new Float64Array( 10 );
+*
+* // Fill the array with pseudorandom numbers:
+* weibull( out.length, [ 2.0 ], 0, [ 5.0 ], 0, out, 1 );
+*/
+function weibull( N, k, sa, lambda, sb, out, so, options ) {
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/main.js]:
```suggestion
function weibull( N, k, sk, lambda, sl, out, so, options ) {
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/main.js [hunk 51]
```diff
@@ -0,0 +1,72 @@
...
+	var rand = prng( k, sa, 0, lambda, sb, 0, arguments.length > 7, options );
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/main.js]:
```suggestion
	var rand = prng( k, sk, 0, lambda, sl, 0, arguments.length > 7, options );
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/main.js [hunk 52]
```diff
@@ -0,0 +1,72 @@
...
+	if ( rand.arity === 0 ) {
+		nullary( [ out ], [ N ], [ so ], rand.fcn );
+		return out;
+	}
+	binary( [ k, lambda, out ], [ N ], [ sa, sb, so ], rand.fcn );
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/main.js]:
```suggestion
	binary( [ k, lambda, out ], [ N ], [ sk, sl, so ], rand.fcn );
```

File: lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js [hunk 53]
```diff
@@ -0,0 +1,75 @@
...
+
+var nullary = require( '@stdlib/strided/base/nullary' ).ndarray;
+var binary = require( '@stdlib/strided/base/binary' ).ndarray;
+var prng = require( './prng.js' );
+
+
+// MAIN //
+
+/**
+* Fills a strided array with pseudorandom numbers drawn from a weibull distribution.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/lib/ndarray.js]:
```suggestion
* Fills a strided array with pseudorandom numbers drawn from a Weibull distribution.
```

**Review by Planeshifter — CHANGES_REQUESTED** [review]

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 54]
```diff
@@ -0,0 +1,175 @@
...
+        Number of indexed elements.
+
+    k: ArrayLikeObject
+        scale parameter.
+
+    sa: integer
+        Index increment for `alpha`.
+
+    lambda: ArrayLikeObject
+        shape parameter.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
        Shape parameter.
```

File: lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt [hunk 55]
```diff
@@ -0,0 +1,175 @@
...
+    If `N` is less than or equal to `0`, the output strided array is left
+    unchanged.
+
+    Parameters
+    ----------
+    N: integer
+        Number of indexed elements.
+
+    k: ArrayLikeObject
+        scale parameter.
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/random/strided/weibull/docs/repl.txt]:
```suggestion
        Scale parameter.
```

**Review by Planeshifter — CHANGES_REQUESTED** [review]

**Comment by amitjimiwal** [comment]
> Thanks for your PR!
> 
> Flagged a bunch of copy-paste and minor naming issues. Once these are addressed (easiest may be to batch commit the suggestions from the files tab), should be able to merge in.

Will do the required changes. Thanks for the review @Planeshifter

**Comment by amitjimiwal** [comment]
@Planeshifter   Requested Chnages are resolved.
Pls review

**Comment by Planeshifter** [comment]
Thank you. There were a few more copy-paste errors to be resolve (see above commits), but your PR has now landed!

**Comment by amitjimiwal** [comment]
Thanks @Planeshifter .

**Pull request merged**